bp-templates/bp-nouveau/includes/groups/template-tags.php

```diff
@@ -1309,58 +1294,47 @@ function bp_nouveau_groups_get_customizer_widgets_link() {
  *
  * @since 1.0.0
  *
- * @param object|bool  $group  Optional. The group being referenced.
-	*                             Defaults to the group currently being
-	*                             iterated on in the groups loop.
-	* @param int          $length Optional. Length of returned string, including ellipsis.
-	*                             Default: 225.
-	* @return string Excerpt.
+ * @param object $group  Optional. The group being referenced.
+ *                       Defaults to the group currently being
+ *                       iterated on in the groups loop.
+ * @param int $length    Optional. Length of returned string, including ellipsis.
+ *                       Default: 100.
+ *
+ * @return string Excerpt.
  *
  */
-function bp_nouveau_group_description_excerpt( $group = false, $length = false) {
-	echo bp_nouveau_get_group_description_excerpt( $group, $length);
+function bp_nouveau_group_description_excerpt( $group = null, $length = 100 ) {
+	echo bp_nouveau_get_group_description_excerpt( $group, $length );
 }
 
 /**
-* Filters the excerpt  of a group description.
-*
-* Checks if the group loop is set as a 'Grid' layout and returns a reduced excerpt.
-*
-* @since 1.0.0
-*
-* @param object|bool  $group  Optional. The group being referenced.
-*                             Defaults to the group currently being
-*                             iterated on in the groups loop.
-* @param int          $length Optional. Length of returned string, including ellipsis.
-*                             Default: 225.
-* @return string Excerpt.
-*/
-function bp_nouveau_get_group_description_excerpt( $group =false, $length = false) {
+ * Filters the excerpt  of a group description.
+ *
+ * Checks if the group loop is set as a 'Grid' layout and returns a reduced excerpt.
+ *
+ * @since 1.0.0
+ *
+ * @param object $group  Optional. The group being referenced.
+ *                       Defaults to the group currently being
+ *                       iterated on in the groups loop.
+ * @param int $length    Optional. Length of returned string, including ellipsis.
+ *                       Default: 100.
+ * @return string Excerpt.
+ */
+function bp_nouveau_get_group_description_excerpt( $group = null, $length = 100 ) {
 	global $groups_template;
 
-	if ( empty( $group ) ) {
+	if ( ! $group ) {
 		$group =& $groups_template->group;
 	}
 
-	// If this is a grid layout but no length is passed in set shorter
-	// default value otherwise use the passed in value.
-	// If not a grid then the BP core default is used or passed in value.
-	if ( bp_nouveau_loop_is_grid() && 'groups' === bp_current_component() ) {
```

Comment:
In defaulting the $length you appear to have done that for all instances of group excerpt, the point of my check was to allow a reduced default where grids are displaying otherwise the boxes break too easily, further my logic allowed for the default to be overridden if $length arg passed.